x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/analysis/analyzer/VerifierErrorMessagesTests.java

```diff
@@ -372,72 +372,72 @@ public void testInNestedWithDifferentDataTypesFromLeftValue_WhereClause() {
             error("SELECT * FROM test WHERE int = 1 OR text IN (1, 2)"));
     }
 
-    public void testNotSupportedAggregateOnDate() {
-        assertEquals("1:8: [AVG(date)] argument must be [numeric], found value [date] type [date]",
+    public void testNotSupportedAggregateOnDateTime() {
+        assertEquals("1:8: [AVG(date)] argument must be [NUMERIC], found value [date] type [DATETIME]",
             error("SELECT AVG(date) FROM test"));
     }
 
     public void testNotSupportedAggregateOnString() {
-        assertEquals("1:8: [MAX(keyword)] argument must be [numeric or date], found value [keyword] type [keyword]",
+        assertEquals("1:8: [MAX(keyword)] argument must be [NUMERIC or DATETIME], found value [keyword] type [KEYWORD]",
```

Comment:
What's with the uppercasing of the types? It is used as lowercase since that's the convention used by ES itself.